subsys/bluetooth/mesh/provisioner.c

```diff
@@ -874,6 +942,11 @@ int bt_mesh_pb_remote_open_node(struct bt_mesh_rpr_cli *cli,
 		return -ENOENT;
 	}
 
+	/* Check if server is on same device as client */
+	if (IS_ENABLED(CONFIG_BT_MESH_RPR_SRV) && bt_mesh_has_addr(srv->addr)) {
```

Comment:
Does this work when `CONFIG_BT_MESH_RPR_SRV` is disabled? I think it won't compile as `reprovision_local_client_server` under `ifdef`.

Reply:
ifdef removed.